content/working_with/console/pages/_index.md

```diff
@@ -22,42 +22,62 @@ Below are the default set of pages (templates) presented per {{< param product_n
 ***main-sys_admin*** template applies to `sys-admin` users in all the tenants, and includes the following pages:
 
 * **Dashboard** - An overview of the existing executions and system statistics.
```

Comment:
Optional:

Regarding the description of the pages content - as it main page for all pages, maybe worth to make all the referenced Cloudify resources here linked to the pages describing what are they?

Reply:
Done